app/javascript/App/bundles/NewsfeedPage/NewsListItem.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react'
+import { Link } from 'react-router-dom'
+import timeago from 'timeago.js'
+import CoinTags from '../../components/CoinTags'
+import BulletSpacer from '../../components/BulletSpacer'
+import { slugify } from '../../lib/utils/slugify';
+
+const twitterLogo = require('../../images/logo-twitter.svg');
+const linkLogo = require('../../images/logo-link.svg');
+const redditLogo = require('../../images/logo-reddit.svg');
+
+const readNewsHandler = (newsItem) => {
+  const newsId = newsItem.id;
+  const readNewsData = JSON.parse(localStorage.getItem('readNews')) || []
+
+  readNewsData.push(newsId)
+  localStorage.setItem('readNews', JSON.stringify(readNewsData))
+
+  if (
+    document.querySelector('.selected-news-content') &&
+    document.querySelector('.selected-news-content').parentNode
+  ) {
+    // @ts-ignore
+    document.querySelector('.selected-news-content').parentNode.scrollTop = 0
+  }
+}
```

Comment:
I don't have enough domain knowledge to understand what's going on here: why not using react environment here (refs etc)?